src/cmake/cuda_macros.cmake

```diff
@@ -95,8 +95,8 @@ FUNCTION ( LLVM_COMPILE_CUDA llvm_src headers prefix llvm_bc_cpp_generated extra
     MAKE_CUDA_BITCODE (${llvm_src} "" llvm_bc "${extra_clang_args}")
 
     ADD_CUSTOM_COMMAND (OUTPUT ${llvm_bc_cpp}
-        COMMAND "${CMAKE_SOURCE_DIR}/src/liboslexec/serialize-bc.bash" ${llvm_bc} ${llvm_bc_cpp} ${prefix}
+        COMMAND "${CMAKE_SOURCE_DIR}/src/liboslexec/serialize-bc.py" ${llvm_bc} ${llvm_bc_cpp} ${prefix}
```

Comment:
If you chmod+x the .py file, that should be good enough on Unix, dunno about Windows, I'm always unsure if the `#!/usr/bin/env` trick will work there.

Is it better to just take no chances and make this line

    COMMAND python "${CMAKE_SOURCE_DIR}/src/liboslexec/serialize-bc.py" ${llvm_bc} ${llvm_bc_cpp} ${prefix}

??

Reply:
Pushed your second suggestion which appears to have worked great! Thanks